apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/TempAuthMockTest.java

```diff
@@ -45,13 +46,24 @@
 
 
    public void testTempAuthRequest() throws Exception {
-      tempAuthServer.enqueue(new MockResponse().setResponseCode(204)
+     Properties overrides = null;
+     // with default values
+     test(overrides);
+     overrides = new Properties();
+     overrides.setProperty(SwiftAuthenticationModule.TEMP_AUTH_HEADER_USER , "X-Auth-User");
+     overrides.setProperty(SwiftAuthenticationModule.TEMP_AUTH_HEADER_PASS , "X-Auth-Pass");
+     // with specific Header Name values
+     test(overrides);
```

Comment:
Better move this to another test named `testTempAuthRequestWithCustomHeaders` or similar, to properly qualify and isolate the test that verifies this change.